eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/SubmittedVariantsRestController.java

```diff
@@ -41,8 +42,10 @@ public SubmittedVariantsRestController(BasicRestController basicRestController)
 
     @ApiOperation(value = "Find submitted variants by identifier")
     @GetMapping(value = "/{identifiers}", produces = "application/json")
-    public Map<Long, SubmittedVariantDTO> get(@PathVariable List<Long> identifiers) {
-        return basicRestController.get(identifiers);
+    public List<AccessionResponseDTO> get(@PathVariable List<Long> identifiers,
+                                          @RequestParam(name = "hideDeprecated", required = false,
+                                                        defaultValue = "false") boolean hideDeprecated) {
```

Comment:
@cyenyxe we were not sure what interface should we provide here about the `hideDeprecated` parameter, what do you think?